packages/dds/tree/src/tree/cursor.ts

```diff
@@ -166,7 +166,7 @@ export interface ITreeCursor {
     // ********** APIs for when mode = Nodes and not pending ********** //
 
     /**
-     * Enters the first field (setting mode to `Fields`)
+     * Enters the first non-empty field (setting mode to `Fields`)
```

Comment:
nextField (above) is needs to updated.

it should note that it goes to the next field, skipping any empty fields.